wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -67,6 +67,7 @@ def breadcrumbs(
     page_perms=None,
     querystring_value=None,
     trailing_breadcrumb_title=None,
+    classes=None,
```

Comment:
To avoid more inconsistent appropriate to adding classes to template tags / components could we opt for the most common approach that already exists?

```suggestion
    classname=None,
```

See https://github.com/wagtail/wagtail/pull/8886
See https://github.com/wagtail/wagtail/issues/8885

Reply:
Oops great catch!! I've added those suggestions now 😄

Reply:
Thanks for doing that. Sorry to be pedantic about this kind of thing, appreciate the fix.